modules/exploits/linux/http/vestacp_exec.rb

```diff
@@ -0,0 +1,242 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::Ftp
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Vesta Control Panel Authenticated Remote Code Execution",
+      'Description'    => %q{
+        This module exploits command injection vulnerability in v-list-user-backups bash script file.
+        Low privileged authenticated users can execute arbitrary commands under the context of the root user.
+
+        An authenticated attacker with a low privileges can inject a payload in the file name starts with dot.
+        During the user backup process, this file name will be evaluated by the v-backup-user bash scripts. As
```

Comment:
Can we change this from `scripts` to `script`?  As far as I am aware there is only one `v-backup-user` script on the system, but feel free to correct me if I am wrong. Thanks for renaming this though, much appreciated!